apps/hash-external-services/kratos/kratos.dev.yml

```diff
@@ -53,7 +53,10 @@ selfservice:
           hooks:
             - hook: web_hook
               config:
-                can_interrupt: true
```

Comment:
I think this may have been added in the hopes to somehow interrupt the flow if an error occurs in the webhook. Not sure what exactly kratos would've done in this circumstance with the field set to try, but if the field is being deprecated anyway we can worry about handling this properly down the line.